src/components/feed/detail/DetailFeedCard.tsx

```
@@ -237,6 +238,20 @@ const Name = styled(Text)`
   }
 `;
 
+interface Vote {
+  id: number;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Vote'. Consider to delete it or rename it.

'Vote' is defined here:


(lint/suspicious/noRedeclare)